packages/block-editor/src/components/url-popover/test/index.js

```diff
@@ -11,6 +11,16 @@ import URLPopover from '../';
 
 jest.useRealTimers();
 
+/**
+ * Returns the first found popover element up the DOM tree.
+ *
+ * @param {HTMLElement} element Element to start with.
+ * @return {HTMLElement|null} Popover element, or `null` if not found.
+ */
+function getWrappingPopoverElement( element ) {
```

Comment:
Maybe it's time to finally extract this to some common helper? 🙂

Reply:
I agree 👍